internal/testing/pruning_test.go

```diff
@@ -259,15 +262,19 @@ func TestSectorPruning(t *testing.T) {
 		t.Fatal("expected 0 prunable data", n)
 	}
 
-	// delete the object
-	if err := b.DeleteObject(context.Background(), "obj", false); err != nil {
-		t.Fatal(err)
+	// delete a random number of objects
+	toDelete := frand.Uint64n(10)
```

Comment:
This might be 0. I think instead of doing it randomly we could also delete every other object here. Randomness in tests can be very annoying.